### DISCUSSION TITLE
Gruntwork Pipelines and infra-live as subdirectory

### DISCUSSION THREAD

**Discussion by apweisskopf** [gruntwork-io/knowledge-base#802] [How do I]
We are using something between a v1 and v2 Gruntwork Reference Architecture and have `infrastructure-live` and `infrastructure-modules` in the same repository:

```
infrastructure.git/modules         <- corresponds to infrastructure-modules
infrastructure.git/environments    <- corresponds to infrastructure-live
```

Our aim is to use the new Gruntwork Pipelines with GitHub Actions.

We followed the hello world example from https://docs.gruntwork.io/pipelines/hello-world

Setting up a new and separate `infrastructure-pipelines` repository was no problem. We then merged the required files of the bootstrapped `infrastructure-live` repo into our existing repo. But when creating a PR and finally triggering the `pipelines` cli to create the change set, it failed:

Pipelines Orchestrate
```
JOBS=$(pipelines orchestrate \
  --repository . \
  --repository-remote-url https://github.com/myorg/infrastructure \
  --source-ref origin/main \
  --target-ref c8ab2ec8529de83c6e9c3118835548d6f22faa25 \
  --event-type pr-synched-created | jq .)
echo $JOBS
echo "jobs=$(echo $JOBS)" >> $GITHUB_OUTPUT

time="2024-01-29T20:07:44Z" level=error msg="No accounts in accounts.yml" binary=pipelines version=v0.2.0
```

We do have an `accounts.yml`, but it is not in the repository root as expected but in `environments/accounts.yml`

How do I tell the pipelines that my `infrastructure-live` code is in a subdirectory?

**Comment by iantybo** [comment]
Following up with a quick answer: Gruntwork Pipelines rely heavily on a set folder structure, with accounts.yml being at the root of a repository and the account folders.
